tools/travis/template_dm_generator.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+import os.path
```

Comment:
redundant line

Reply:
same as tg

Reply:
Seems to be redundant from what it's said, just accept the suggestion.

Reply:
> 
> 
> Seems to be redundant from what it's said, just accept the suggestion.

i just mean i'll change it upstream